src/core/namespace.c

```diff
@@ -1073,6 +1074,19 @@ static int mount_sysfs(const MountEntry *m) {
         return 1;
 }
 
+static bool mount_option_supported(const char *fstype, const char *key, const char *value) {
+        _cleanup_close_ int fd = -1;
+
+        /* This function assumes support by default. Only if the fsconfig() call fails with -EINVAL/-EOPNOTSUPP
+         * will it report that the option/value is not supported. */
+
+        fd = fsopen(fstype, FSOPEN_CLOEXEC);
+        if (fd < 0)
+                return true; /* If fsopen() fails for whatever reason, assume the value is supported. */
```

Comment:
maybe log_debug() about that failure?

Reply:
(i.e. about unexpected failures at the very least, but might even make sens to log about all cases here)